docs/receiving.md

```diff
@@ -1,107 +1,117 @@
 ---
 id: receiving
-title: Receiving Payments
+title: Web Monetization Receivers
 sidebar_label: Receiving Payments
 ---
 
-When a user has Web Monetization (WM) enabled in their browser and they visit a
-monetized website, the browser resolves a unique receiving address for payments
-to the website's WM receiver.
+This page describes the functions of the Web Monetization receiver and how
+existing payment services might become a receiver.
 
-The browser then begins sending payments to the website's receiving address
-through the user's WM Sender.
+A receiver is an entity that accepts payments on behalf of a website and is
```

Comment:
A Web Monetization Receiver rather than a Receiver

"A web monetization receiver is an entity that accepts payments over interledger" rather than just "accepts payments"?